datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxRepairService.java

```diff
@@ -133,10 +132,8 @@ private ClusterRepairV4Request createRepairRequest(SdxRepairSettings sdxRepairSe
     }
 
     public void waitCloudbreakClusterRepair(Long id, PollingConfig pollingConfig) {
-        sdxClusterRepository.findById(id).ifPresentOrElse(sdxCluster -> {
-            cloudbreakPoller.pollUpdateUntilAvailable("Repair", sdxCluster, pollingConfig);
-            sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.RUNNING, ResourceEvent.SDX_REPAIR_FINISHED, "Datalake is running", sdxCluster);
-        }, () -> {
+        sdxClusterRepository.findById(id).ifPresentOrElse(sdxCluster ->
+            cloudbreakPoller.pollUpdateUntilAvailable("Repair", sdxCluster, pollingConfig), () -> {
```

Comment:
This seemingly random change is because I found out the repair service had duplicate success events (the exact call I removed is called in the flow right after this), so I removed it.

Reply:
Please revert changes not relevant to this PR.